### PR TITLE
Sentinel 5P to panoply

### PR DESCRIPTION
**Pull request by Marie59**
Turorial on how to download Sentinel 5P with a Copernicus openeo notebook and then visualise those data with Panoply.

1. Give your pull request a **good title**
    - e.g. "Fix typo in ansible-galaxy tutorial" or "Add new transcriptomics tutorial covering a new sequencing technology"
2. Describe your changes in detail, list anything you still need some help with or things that are still TODO
3. Check that your images are allowed to be re-hosted by the GTN!
4. Still working on it? Make it a **Draft** pull request
   - Once it is ready, choose **Ready for Review**
   - Then the automated tests will run and we will know to review and merge it

### CODE REVIEW

File: topics/climate/tutorials/sentinel5_data/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  - satellite data
+contributions:
+  authorship:
+    - Marie59
+  funding:
+    - fairease
+    - eurosciencegateway
+---
+
+# Introduction
```
Comment (hexylena) [inline topics/climate/tutorials/sentinel5_data/tutorial.md:32]:
```suggestion
```

File: topics/climate/tutorials/sentinel5_data/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+Through this tutorial you will learn here how to access and download Copernicus Data Space Ecosystem (CDSE) data through a jupyterlab Galaxy interactive tool :
+
+This tool enables you to leverage the Copernicus Data Space Ecosystem services and access data effortlessly.
+The JupyterLab service allows you to dive into data exploration, visualization, and analysis without the hassle of installing dependencies or downloading large data sets.
+
+Then, you can visualise these data and process them with Panoply (plots geo-referenced and other arrays from netCDF, HDF, GRIB, and other datasets).
+
+The purpose of this tutorial is to propose a user-friendly, interactive and efficient way to explore and jointly analyze remote sensing observations, from both the atmosphere and solid earth communities, for the monitoring of the volcanic activity in case of eruption and the multi-scale impact of volcanic emissions on the atmosphere.
+
+> <details-title>What about the data used here/details-title>
```
Comment (hexylena) [inline topics/climate/tutorials/sentinel5_data/tutorial.md:44]:
```suggestion
> <details-title>What about the data used here</details-title>
```

File: topics/climate/tutorials/sentinel5_data/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+> ![Begining of the Sentinel 5 notebook  with the connection link visible](../../images/sentinel_5/notebook_begin.png)
+> 7. Once you clicked a new window opens. If you don't have a copernicus account please create one and follow the instructions given. 
+> 8. If you have an account sign in and then press the green **YES**
+> ![Image of how does the connection page looks like](../../images/sentinel_5/connection.png)
+> 9. You'll be redirected to a page letting you know the connection was a success
+> ![Image of the successful connection](../../images/sentinel_5/connection_ok.png)
+> 10. Now, you can go back on your jupyterlab and execute the rest of the notebook.
+> 11. Once you executed the entire notebook you should have a new file in the left pannel named **sentinel5.nc**.
+> 12. Select the file there and click right, and then copy
+> ![The copy command on the 3 netcdf files](../../images/sentinel_5/copy.png)
```
Comment (hexylena) [inline topics/climate/tutorials/sentinel5_data/tutorial.md:118]:
```suggestion
>      ![The copy command on the 3 netcdf files](../../images/sentinel_5/copy.png)
```

**Comment by Marie59** [comment]
The last thing not going through is the test-data part.

**Review by hexylena — APPROVED** [review]

**Comment by Marie59** [comment]
Thank you @hexylena !!

**Pull request merged**